packages/meditrak-server/src/apiV2/deleteAccount.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+import { respond } from '@tupaia/utils';
+import { sendEmail } from '../utilities';
+import { getUserInfoInString } from './utilities';
+
+const sendRequest = userName => {
+  const { TUPAIA_ADMIN_EMAIL_ADDRESS } = process.env;
+
+  const emailText = `${userName} has requested to delete his/her account`;
```

Comment:
```suggestion
  const emailText = `${userName} has requested to delete their account`;
```
? Tiny but reads a tad clearer